### ISSUE TITLE
Save as template -functionality

### ISSUE THREAD

**Issue by robotframework-ride** [robotframework/RIDE#213]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=178) by kahvimugi on 25 Jun 2009*

If you want to create Test Suite templates with RIDE, it should be 
possible to save Test Suite files 'as its own template'. 
Currently, if the user wants to make any file a template, he/she must add 
&lt;meta name="rf-template" content="True" /&gt; inside head -tag in the HTML 
source manually - RIDE could do this easilly for him/her when wanted.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=178#c1) by @pekkaklarck on 25 Jun 2009*


<a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=179" title="Save as template -functionality"> Issue 179 </a> has been merged into this issue.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=178#c4) by @yanne on 5 Sep 2010*


We should probably change the template value to be true by default, and also offer an option to clean up external content from the file.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=178#c7) by @Tattoo on 6 Nov 2012*


HTML templates are broken; blocked by <a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=1097" title="HTML templates are broken"> issue 1097 </a>

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=178#c8) by @mkorpela on 19 Dec 2012*


HTML templates is a discontinued feature before 0.40.
